### PR TITLE
Fix user model

### PR DESCRIPTION
**Pull request by 2HEADSs**
change default values of isDeleted and isApproved values to handle a bug.
The bug was that when someone deleteD "isApproved" or "isDeleted" properties in DB of user, when the same user create event or login in the app, mongo automaticaly set "isDeleted" and "isApproved" with their default value. 
For that reason I change the default values:
  isDeleted: { type: Boolean, default: true },
  isApproved: { type: Boolean, default: false },
  and set them in register controller

### CODE REVIEW

File: src/controllers/userController.js [hunk 1]
```diff
@@ -56,14 +56,20 @@ userController.post('/register', async (req, res) => {
       role: req.body.role ? req.body.role : 'regular',
       region: req.body.region ? req.body.region : '',
       phone: req.body.phone ? req.body.phone : '',
+      isDeleted: false,
       hashedPassword: await bcrypt.hash(req.body.password, 10),
     };
+    if (userData.role == 'regular') {
+      userData.isApproved = true;
```
Comment (Getsov) [inline src/controllers/userController.js:63]: And what about admins?

File: src/models/User.js [hunk 2]
```diff
@@ -90,8 +90,8 @@ const userSchema = new Schema({
   createdEvents: [{ type: ObjectId, ref: 'Event' }],
   likedEvents: [{ type: ObjectId, ref: 'Event' }],
   hashedPassword: { type: String, required: true, select: false },
-  isDeleted: { type: Boolean, default: false },
-  isApproved: { type: Boolean, default: true },
+  isDeleted: { type: Boolean, default: true },
```
Comment (Getsov) [inline src/models/User.js:93]: Default must be false

File: README.md [hunk 3]
```diff
@@ -21,11 +21,12 @@ The service is not initialized with any data. You need to create it by yourself.
...
+Create a new `user` by sending a POST request to `http://localhost:3030`: + `/user/register` with properties: `email`, `password`, `repassword`.If you register as admin, you need to pass `role`. If you register as organizator you need to add to request `role`, `organizatorName`, `phone`. The service automatically creates a session and returns object with `accessToken` inside, that can be used for authorized requests.
+Admins and organizаtors will be active after approval by active admin.
 
 ### Login:
 
-Login by sending a POST request with `email` and `password` to `http://localhost:3030` + `/user/login`. The service will respond with object with details containing a standard `accessToken`, that can be used for authorized requests.
+Login by sending a POST request with `email` and `password` to `http://localhost:3030` + `/user/login`. If the profile is approved and it's not deleted, the service will respond with object with details containing a standard `accessToken`,that can be used for authorized requests.
```
Comment (Getsov) [inline README.md:29]: and is not deleted*

File: src/models/User.js (hunk 2 above)
Comment (2HEADSs) [reply src/models/User.js]: fixed

File: src/controllers/userController.js (hunk 1 above)
Comment (2HEADSs) [reply src/controllers/userController.js:60]: the default value is false and will be waiting approval.

File: README.md (hunk 3 above)
Comment (2HEADSs) [reply README.md]: fixed

**Review by Getsov — APPROVED** [review]

**Pull request merged**